clinica/iotools/converters/ixi_to_bids/ixi_to_bids_utils.py

```diff
@@ -0,0 +1,209 @@
+import json
+import re
+import shutil
+from pathlib import Path
+from typing import List, Optional
+
+import nibabel as nib
+import pandas as pd
+from nilearn.image import concat_imgs
+
+from clinica.iotools.bids_utils import StudyName, bids_id_factory
+from clinica.utils.stream import cprint
+
+
```

Comment:
Here it would be great to know what are the functions that are exposed to the outside and what are the internal private functions. You can prefix all private functions with "_", and list all exported objects in `__all__`.

Moreover, for exported functions, it would be nice to have proper docstrings with the description of parameters, and return values.